lib/util/model/ModelCloneUtils.js

```diff
@@ -9,6 +9,7 @@ import {
  * @type {Array}
  */
 export var IGNORED_PROPERTIES = [
+  'host',
```

Comment:
What is this about?

Reply:
I thought it might be useful in the context of host update, but actually we never use this. I'll remove it.